packages/cli/CHANGELOG.md

```diff
@@ -1,10 +1,10 @@
 # Changelog
 
-## Unreleased
-* Fix AMD transform bug where if an HTML document had multiple type=module scripts, and any of them (apart from the first) had any kind of import, then that import was not accessible (because it was mapped to the wrong module callback function argument).
+<!-- ## Unreleased -->
 <!-- Add new, unreleased items here. -->
 
 ## v1.7.0-pre.9 [04-11-2018]
+* Fix AMD transform bug where if an HTML document had multiple type=module scripts, and any of them (apart from the first) had any kind of import, then that import was not accessible (because it was mapped to the wrong module callback function argument).
```

Comment:
Note this release was going to go out earlier, but I wanted to include this recent fix, hence no new heading.